app/sms/application_sms.rb

```diff
@@ -0,0 +1,11 @@
+class ApplicationSms
+  attr_reader :client, :application_number
+
+  def initialize(*)
```

Comment:
alternative to `*args` when one wants to allow any args and not use them specifically (in this case, only subclasses accept args). rubocop likes it better